pages/productAppExtension/[productId]/index.tsx

```diff
@@ -0,0 +1,39 @@
+import { H4, Panel, Text } from "@bigcommerce/big-design";
+import { useRouter } from "next/router";
+import ErrorMessage from "@components/error";
+import Loading from "@components/loading";
+import { useProductInfo } from "@lib/hooks";
+
+const ProductAppExtension = () => {
+    const router = useRouter();
+    const productId = Number(router.query?.productId);
+    const list = [];
+    const { error, isLoading, product } = useProductInfo(productId, list);
+    const { description, is_visible: isVisible, name, price, type } = product ?? {};   
+    const typeCapitalized = type?.replace(/^\w/, (c: string) => c.toUpperCase());
+    const isVisibleString = isVisible?.toString().replace(/^\w/, (c: string) => c.toUpperCase());
```

Comment:
🍹 this seems a bit complicated for what we're trying to do... I'd recommend either moving out the function to capitalize to a helper function or maybe just making this an easy ternary:

```suggestion
    const isVisibleString = isVisible ? 'True' : 'False';
```